src/components/ArticleCard/ArticleCard.js

```diff
@@ -10,8 +10,8 @@ import './ArticleCard.scss';
 let articlePic = '';
 
 export const ArticleCard = (props) => {
-  if (!props.pic) {
-    articlePic = 'https://via.placeholder.com/300?text=AuthorsHaven';
+  if (props.pic === 'null') {
```

Comment:
I did this check because I noticed that the null returned is a string and checking with `!props.pic `does not work as intended